src/pokemon_summary_screen.c

```diff
@@ -3708,7 +3708,7 @@ static void PrintMoveDetails(u16 move)
     {
         if (sMonSummaryScreen->currPageIndex == PSS_PAGE_BATTLE_MOVES)
         {
-            ShowSplitIcon(gBattleMoves[move].split);
+            ShowSplitIcon(GetBattleMoveSplit(move));
```

Comment:
Question: should this show an icon if the split is disabled? It is more useful to the player if the icon is there, but it is more faithful to Gen 3 if the icon is disabled along with the split.

Reply:
Added option to hide the split icon independent from the split mechanics.